pandas/_testing/__init__.py

```diff
@@ -441,7 +442,10 @@ def makeTimedeltaIndex(
 
 def makePeriodIndex(k: int = 10, name=None, **kwargs) -> PeriodIndex:
     dt = datetime(2000, 1, 1)
-    return pd.period_range(start=dt, periods=k, freq="B", name=name, **kwargs)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        pi = pd.period_range(start=dt, periods=k, freq="B", name=name, **kwargs)
```

Comment:
Is it possible to use another freq like "D"?

Reply:
Doing that breaks a handful of tests.  I could go either way on changing it now vs after enforcement.

Reply:
Would slightly prefer changing now to hopefully prevent potentially having to change more tests in the future

Reply:
Fair enough, updated